.github/workflows/auto_create_version_bump_pr.yml

```diff
@@ -0,0 +1,82 @@
+name: Auto version bump on release completed
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  bump_version:
+    # If this is a 1.x release, do the version bump in the release/1.x branch
+    # If this is a 2.x.0 release, do the version bump on master
+    # If this is a 2.x.x hotfix release, _don't_ do a version bump
+    if: |
+      startsWith(github.event.release.tag_name, 'v1.')
+      || (startsWith(github.event.release.tag_name, 'v2.')
+        && endsWith(github.event.release.tag_name, '.0'))
+
+    runs-on: windows-latest
+    env:
+      GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+
+    steps:
+      - name: Support longpaths
+        run: git config --system core.longpaths true
+
+      - name: "Select branch"
+        id: select_branch
+        run: |
+          if( "${{ github.event.release.tag_name}}".StartsWith("v1.")) {
+            echo "::set-output name=ref::release/1.x"
+          } else {
+            echo "::set-output name=ref::master"
+          }
+
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ steps.select_branch.outputs.ref }}
+
+      - name: "Configure Git Credentials"
+        run: |
+          git config user.name "${{ github.actor }}"
+          git config user.email "${{ github.actor }}@users.noreply.github.com"
+
+      - uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: '6.0.x'
+
+      - name: "Update Changelog"
+        run: .\tracer\build.ps1 UpdateChangeLog
```

Comment:
Does the changelog also get the draft release note text relating to artifacts, like the following?

```
⚠ 1. Download the NuGet packages for the release from [this link]({artifactsLink}) and upload to nuget.org
```

Reply:
Yes, they would if they haven't been removed! The person doing the release should remove them when publishing, but it makes sense to explicitly remove then in case they haven't been!

Reply:
A bit hacky, but fixed in https://github.com/DataDog/dd-trace-dotnet/pull/2324/commits/264acd443fc8852e8f0da72cfd1b361a6c397943

Reply:
Ohh I misunderstood the code slightly, I was thinking that this would run when a release draft was _created_ not _published_. Thanks for the additional commit, that's a nice-to-have cleanup in case anything gets missed 👍🏼